x-pack/plugins/security/public/management/table_utils.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { css } from '@emotion/react';
+import type { ReactNode } from 'react';
+import React from 'react';
+
+const RemoveButtonLabelInActionsCellCss = `
+  .euiTableRowCell--hasActions .euiButtonEmpty .euiButtonContent {
+    padding: 0px 0px;
+    .euiButtonEmpty__text {
+        display: none;
+      }
+  }
+`;
+interface ActionsEuiTableFormattingProps {
+  children: ReactNode;
+}
+export const ActionsEuiTableFormatting = React.memo<ActionsEuiTableFormattingProps>(
```

Comment:
Notes for reviewer:  Why did Xavier decide to override EUI? is he crazy?
Answer: maybe he is, but the real reason is that I/we wanted to keep our icon working with onClick and href at the same time but the actions props in `EuiTable` does not let you do that and I still wanted the same style how EUI do it and that's my reason.
if you know/have a better way please let me know, I will love/do it.

Reply:
There isn't a better way (that I'm aware of) to achieve this without changing some things in EUI 👍